atlas_admin/templates/instances/sitestats.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block title %}{%- if themeName -%}{{ themeName }}{%- elif nodeCount -%}{{ nodeCount }}{%-  else -%}All{% endif %}
+instances{% endblock %}
+
+{% block content %}
+{%- if instanceList -%}
+<div class="row">
+    <div class="columns twelve">
+        <table>
+            <th>Name</th>
+            {% for (i, j) in instanceList %}
+            <tr>
+                <td><a href="{{ url_for('.index') }}instances/{{ i }}">{{ j }}</a></td>
```

Comment:
Is there another piece of information that we can add here to help separate all the instances named "University of Colorado Boulder"?

Reply:
Possible identifying information:
1) Days Since Last Edit will show how active the site is
2) Site Owners will show who it belongs to.

Reply:
Both of those sound great, let's start with Site owners

Reply:
Actually it's pretty easy to include both, AND only have that info on sites named University of Colorado Boulder.

Reply:
Feel free to include both, but I would keep the output the same for all results